src/abstractdataframe/join.jl

```diff
@@ -247,6 +247,11 @@ Join two `DataFrame` objects
   if `true`, duplicate names will be suffixed with `_i`
   (`i` starting at 1 for the first duplicate).
 
+* `indicator` : Default: `nothing`. If a `Symbol`, adds categorical indicator
+   column named `Symbol` for whether a row appeared in only `df1` (`left_only`),
+   only `df2` (`right_only`) or in both (`both`). If `Symbol` is already in use,
+   the column name will be modified if `makeunique=true`.
```

Comment:
So what happens when `makeunique=false`? Can we check that explicitly and throw an error if the name is already used?

Reply:
if `makeunique=false`, it ignores it and just acts like `makeunique=true`. Again, see #1425

Reply:
```
julia> a = DataFrame(col = [1, 2, 3])
3×1 DataFrame
│ Row │ col │
├─────┼─────┤
│ 1   │ 1   │
│ 2   │ 2   │
│ 3   │ 3   │

julia> b = DataFrame(col = [1, 2, 3])
3×1 DataFrame
│ Row │ col │
├─────┼─────┤
│ 1   │ 1   │
│ 2   │ 2   │
│ 3   │ 3   │

julia> hcat(a, b, makeunique=false)
3×2 DataFrame
│ Row │ col │ col_1 │
├─────┼─────┼───────┤
│ 1   │ 1   │ 1     │
│ 2   │ 2   │ 2     │
│ 3   │ 3   │ 3     │
```

Reply:
OK. But I guess a deprecation warning is printed? If not, better throw an error manually.

Reply:
The warning is printed and an error will be thrown after deprecation period.

Reply:
OK, let's say that's fine then.